api/python/quilt3/data_transfer.py

```diff
@@ -19,6 +20,7 @@
 from botocore.exceptions import ClientError, ConnectionError, HTTPClientError, ReadTimeoutError
 import boto3
 from boto3.s3.transfer import TransferConfig
+from quilt3 import util
```

Comment:
Quilt imports should go below third party ones. Also, just import individual functions; see the `from .util import ...` below